src/index.js

```diff
@@ -220,7 +225,7 @@ function sectionUrlComponent(value, key) {
       else if (!(/http/i.test(url))) target_url = 'http://' + url
 
       sectionHTML = replaceAll(sectionHTML, url, `<a href="${target_url}" target="_blank" onclick="captureOutboundLink('${target_url}', '${key}')">${url}</a>`);
-    })
+    });
```

Comment:
I ... might be tempted to add prettier to this ahead of the hackathon :D

Reply:
Yeah.. I was really struggling with that throughout this but decided not to because I thought it would make the PR a lot larger and maybe be confusing for reviewers. Might be good to have a PR in and of itself that just prettifies and doesn't make any real code changes.

Reply:
Totally.